ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_pool.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# author alok ranjan (alok.ranjan2@hpe.com)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+author:
+  - Alok Ranjan (@ar-india)
+description: Manage storage pools on HPE Nimble Storage group.
+module: hpe_nimble_pool
+options:
+  array_list:
+    required: False
+    type: list
+    description:
+    - List of arrays in the pool with detailed information. When create/update array list, only arrays' ID is required.
```

Comment:
nit: List of arrays in the pool with detailed information. To create or update array list, only array IDs are required.